src/resources/postcss/dialog.pcss

```diff
@@ -184,4 +184,38 @@ Adding these here for now until common-styles are live
 		justify-content: flex-end;
 	}
 
+	/* -------------------------------------------------------------------------
+	 * Theme Overrides - Avada
+	 * ------------------------------------------------------------------------- */
+
+	 .tribe-theme-avada & {
+
+		div.tribe-dialog {
+			z-index: 99999;
```

Comment:
Can we get a variable for that? Possibly something related to theme compatibility.

Reply:
Thanks! We normally don't use variables in the pcss for the theme compatibility, because they're normally overrides. Do you feel we should do an exception here and having different from the rest?

Reply:
Knowing that we've had some issues with variable replacement in the build (for compatibility, as done in TEC) it should be fine to skip the variables here.

That would be a _great_ thing for you to look into though @juanfra - see if you can figure out what the problem is (separate from this)?

Reply:
I really would love to avoid arbitrary numbers. What were the problems with the building?

Since now Tribe Common Styles is being used on Tickets too we can import variables from there at any time correct?

Maybe a variable for Theme compatibility overall? So if in the future we need to bump it, it applies to all themes.

Reply:
In the end it's a `z-index` race. What we're doing here is making sure that we have a higher one for the themes we provide compatibility with. Given that they use arbitrary numbers, we have to use arbitrary numbers to add their compatibility. These are "special cases", and we don't normally set variables for these special cases. There's no silver bullet for theme compatibility, each theme can have whatever `z-index` they want for their header, footer, or the element they want. We ended up giving compatibility with the same themes we have for V2. And if somebody else comes with something, then we can provide a CSS snippet.